pkg/controller/manager/sync.go

```diff
@@ -108,6 +108,10 @@ func (c *Controller) sync(ctx context.Context, key string) error {
 		return nil
 	}
 
+	if helpers.IsStatusConditionPresentAndTrue(sc.Status.Conditions, scyllav1.ProgressingCondition, sc.Generation) {
```

Comment:
It's not enough, you should check all three aggregated conditions

Reply:
What are all the three conditions?

Reply:
https://github.com/scylladb/scylla-operator/blob/6adce4fc649fa41726cecf502b70d9c1cb74cb60/pkg/api/scylla/v1/cluster_types.go#L435

Reply:
Do you think we could have function IsClusterRolledOut like in e2e? If yes, where would be the best dir to place it? Locally for manager controller or somewhere in pkg/util so that all controllers could use it?

Reply:
@zimnx

Reply:
i guess `pkg/controllerhelpers` would be the best place